src/inventory/ModelManager.cpp

```diff
@@ -394,6 +422,50 @@ void ModelManager::scanForModels(QString path) {
     updateAvailableModels();
 }
 
+bool ModelManager::readModelMetaFromDir(ModelMeta &model, QString dir) const {
+    QFile metaFile(dir + "/.modelMeta.json");
+    if (!metaFile.open(QIODevice::ReadOnly | QIODevice::Text)) {
+        qDebug() << "Could not parse model meta file" << metaFile.fileName() << ": file cannot be opened for reading";
```

Comment:
We should remove that, it spits a lot of noise:
`Could not parse model meta file "~/.config/translateLocally/bgen.student.tiny11-1656082614/.modelMeta.json" : file cannot be opened for reading`
If modelMeta is not there, we should generate it to the best of our ability.

Reply:
I don't think there's much value of generating it over just skipping reading it. Trying to generate it could have a whole bunch of error cases on its own, such as read only filesystem, permission issues, etc.

How about instead of removing debug messages, we keep them but hide them behind a `--debug` flag?